scripts/Muon/GUI/ElementalAnalysis/LoadWidget/load_model.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import mantid.simpleapi as mantid
+
+from Muon.GUI.ElementalAnalysis.LoadWidget import load_utils as lutils
+
+
+class LoadModel(lutils.LModel):
+    def __init__(self):
+        super(LoadModel, self).__init__()
+
+    def execute(self):
+        if self.run not in self.loaded_runs:
+            self.load_run()
+
+
+class CoLoadModel(lutils.LModel):
```

Comment:
why have you done it this way?

Reply:
When trying to handle co-loading and loading with one model, the execute function became too large and the model ended up containing almost equal amounts of code for both types of load. I thought it'd be sensible and more maintainable to separate the code out so their specific functions could be isolated

Reply:
sorry, I meant the ``lutils.LModel``

Reply:
Oh, it just contains code common to both, such as the loading and `thread_model`'s required functions - saves rewriting/reusing code